### PR TITLE
Andlrutt/tkw 52 users table

### PR DESCRIPTION
**Pull request by andlrutt**
This PR satisfies TKW-52, creating a `UserTable` with all users from auth0. Almost identical to the `TreeTable` component, but it displays a user's name, email, and phone number. Like with the tree table, the individual rows are clickable and redirect to a specific user page, which will hold all of a user's associated trees in the future. We will also probably replace the "NO PHONE" text for users without a name/phone/email to just be blank, but they are there for testing purposes for now.

Changes: 
- new User type in `/utils/types`
- new paths/api routes in `/utils/urls` for users
- `getUsers()` and `getUser()` server functions
- `SingleUser` component, which comprises the rows of the larger `UserTable` component
- new `src/pages/admin/users/index.tsx` page, which is what holds the `UserTable` component
- new `src/pages/admin/users/[userId].tsx` page, which is what the rows in the table redirect to 
- Cleaned up a slight styling error carried over from the `TreeTable`, screenshot at the end of the PR.

`src/pages/admin/users/index.tsx` page
image

Mobile view
image

`src/pages/admin/users/[userId].tsx` page
image

Previously, because of the way the mobile view was set up, there was a horizontal dividing line on the top of the first table entry, which was slightly visible. This has been removed for desktop view.
Before
image
After
image

### CODE REVIEW

File: server/actions/User.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { Types } from "mongoose";
+import { Tree } from "utils/types";
+import TreeModel from "server/models/Tree";
+
+
+/**
+ * @returns all users in auth0
+ */
+ export const getUsers = async function() {
+  var axios = require("axios").default;
```
Comment (tbatk) [inline server/actions/User.ts]: Use `const` instead of `var`

File: server/actions/User.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  var options = {
```
Comment (tbatk) [inline server/actions/User.ts]: Use `const` instead of `var`

File: server/actions/User.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const response = await axios.request(options)
+  return response.data;
+}
+
+/**
+ * @param id id of a user in auth0
+ * @returns a single user
+ */
+export const getUser = async function(id: string) {
+  var axios = require("axios").default;
```
Comment (tbatk) [inline server/actions/User.ts]: Use `const` instead of `var`

File: server/actions/User.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  var options = {
```
Comment (tbatk) [inline server/actions/User.ts]: Use `const` instead of `var`

File: utils/urls.ts [hunk 5]
```diff
@@ -7,11 +7,16 @@ export default {
         addTree: "/admin/create",
         adminTrees: "/admin/trees",
         updateTree: (treeId: string) => `/admin/trees/${treeId}`,
+        getUser: (userId: string) => `/admin/users/${userId}`
```
Comment (tbatk) [inline utils/urls.ts:10]: Never thought of doing this before. Neat!

**Review by tbatk — CHANGES_REQUESTED** [review]
Good work! There's just a few changes I think that need to be made.

File: utils/urls.ts (hunk 5 above)
Comment (andlrutt) [reply utils/urls.ts:10]: Now that I'm working on TKW-50, I'm probably going to change it back haha.

**Review by tbatk — APPROVED** [review]

**Pull request merged**